contracts/core/Core.sol

```diff
@@ -77,14 +94,23 @@ contract Core is
         _;
     }
 
-    modifier isValidFactoryCheck(address _factoryAddress) {
+    modifier isValidFactory(address _factoryAddress) {
         require(
-            isValidFactory[_factoryAddress],
+            validFactories[_factoryAddress],
             INVALID_FACTORY
         );
         _;
     }
 
+    // Verify set was created by core and is enabled for issuance
```

Comment:
don't mention issuance. i use the same for redeem

Reply:
+1